vivisect/__init__.py

```diff
@@ -2843,6 +2847,14 @@ def loadFromMemory(self, memobj, baseaddr, fmtname=None):
         # Snapin our analysis modules
         self._snapInAnalysisModules()
 
+    def writeMemory(self, va, bytez):
+        '''
+        Override writeMemory to hook into the Event subsystem.
+        Stores overwritten data for easy undo.
+        '''
+        oldbytes = self.readMemory(va, len(bytez))
```

Comment:
Need to implement perms checks here using _reqProbeMem...  it was defined and tested but never used :)

Reply:
done